orchestra/proc-macro/src/parse/parse_orchestra_struct.rs

```diff
@@ -893,3 +921,13 @@ pub(crate) fn consumes_without_wip<'a, T: Borrow<SubSysField>>(subsystems: &[T])
 		.map(|ssf| ssf.message_to_consume())
 		.collect::<Vec<_>>()
 }
+
+pub(crate) fn can_receive_priority_messages_without_wip(
+	subsystems: &Vec<&SubSysField>,
+) -> Vec<TokenStream> {
+	subsystems
+		.iter()
+		.filter(|ssf| !ssf.wip)
+		.map(|ssf| if ssf.can_receive_priority_messages { quote!(true) } else { quote!(false) })
```

Comment:
```suggestion
		.map(|ssf| if syn::LitBool::new(ssf.can_receive_priority_messages, ssf.ident.span()))
```
nit: maintain the span information where possible